doc/markdown.md

```diff
@@ -50,43 +51,62 @@ conventions make it easier to work with lists.
 For **_unordered lists_**, indent all but the first line by **two spaces** more
 than the first line like this (`|` represents the left margin):
 
-    |- First line of bulleted list.
-    |  Second line is indented to match.
-    |
-    |  ```dart
-    |  // Some code
-    |  ```
+````markdown
+|- First line of bulleted list.
+|  Second line is indented to match.
+|
+|  ```dart
+|  // Some code
+|  ```
+````
 
 Start **_ordered lists_ one space** from the left margin, and indent all content
 by **four spaces**:
 
-    | 1. First line of bulleted list.
-    |    Second line is indented to match.
-    |
-    |    ```dart
-    |    // Some code
-    |    ```
+```markdown
+| 1. First line of bulleted list.
+|    Second line is indented to match.
+|
+|    ```dart
+|    // Some code
+|    ```
+```
 
 ## **PREFER** placing link definitions after the text containing them
 
 Embedding link URLs inside text makes the text difficult to read, for example:
 
-    Now that we have a `TabController` to work with, we can create our
-    tabs using the [TabBar](https://docs.flutter.io/flutter/material/TabController-class.html) widget.
-    In this example, we'll create a `TabBar` with 3 [Tab](https://docs.flutter.io/flutter/material/Tab-class.html)
-    widgets and place it within an [AppBar](https://docs.flutter.io/flutter/material/AppBar-class.html).
+```markdown
+Now that we have a `TabController` to work with, we can create our
+tabs using the [TabBar](https://api.flutter.dev/flutter/material/TabBar-class.html) widget.
+In this example, we'll create a `TabBar` with 3 [Tab](https://api.flutter.dev/flutter/material/Tab-class.html)
+widgets and place it within an [AppBar](https://api.flutter.dev/flutter/material/AppBar-class.html).
```

Comment:
Hmmmm. This is actually a bad doc example, as it should use second person, not first: https://developers.google.com/style/person

Reply:
Nice catch, I didn't even read it while updating the links. Thanks!